breakpoints.js

```diff
@@ -27,6 +29,32 @@
 		lastSize = 0;
 	};
 	
+		var scrollbarWidth = 0;
+
+		$.getScrollbarWidth = function() {
+		/* Gets the width of the OS scrollbar		
+		https://github.com/brandonaaron/jquery-getscrollbarwidth/blob/master/jquery.getscrollbarwidth.js */
+		
+			if ( !scrollbarWidth ) {
+				if ( $.browser.msie ) {
```

Comment:
jquery deprecated $.browser with version 1.9
Maybe we can define something like that instead of $.browser.msie:
var isMsie = /MSIE/gi.test(navigator.userAgent);